reconcile/cli.py

```diff
@@ -551,31 +551,41 @@ def github_validator(ctx):
 @integration.command(
     short_help="Configures ClusterRolebindings in OpenShift clusters."
 )
+@click.option('--no-github',
+              is_flag=True,
+              help='use org_username instead of github_username for ' +
+              'user <> clusterrolebinding associations')
 @threaded()
 @binary(['oc', 'ssh'])
 @binary_version('oc', ['version', '--client'], OC_VERSION_REGEX, OC_VERSION)
 @internal()
 @use_jump_host()
 @click.pass_context
-def openshift_clusterrolebindings(ctx, thread_pool_size, internal,
-                                  use_jump_host):
+def openshift_clusterrolebindings(ctx, no_github, thread_pool_size,
```

Comment:
Just a general note that for any new parameters please add types. For anything existing, it doesn't hurt to if you've already figured out what they are. I'll just make this comment once rather than for each instance.